README.md

```diff
@@ -4,13 +4,19 @@
 
 Please visit the infrastructure repo for setting up a local development environment at <https://github.com/CMSgov/blueprint_infra>
 
+## Visual Elements
+This application leverages [USWDS](https://designsystem.digital.gov/) and [Trussworks' React-USWDS](https://github.com/trussworks/react-uswds).  The application currently has the USWDS dependency pinned to version 2.x.x until Trussworks releases a version of it's library for the latest USWDS enhancements.
```

Comment:
Thanks for adding that context for whoever is working on this next. We might want to add something about the `--legacy-peer-deps` part of the `RUN npm install --legacy-peer-deps` in the Dockerfile. I'm pretty sure that was added in because of Trussworks but I don't know enough about the reason it was needed.